tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -8,25 +8,89 @@ Suite Setup       user signs in as bau1
 Suite Teardown    user closes the browser
 
 *** Variables ***
-${DETAILS_HEADING}  Academic Year 2020/21
+${DETAILS_HEADING}  Academic Year Q1 2020/21
 ${PUBLICATION_NAME}  Publish Release and Amend 2 Publication %{RUN_IDENTIFIER}
-${SUBJECT_NAME}  Dates test subject
+${SUBJECT_NAME}  SevenFiltersTestSubject
 ${SECOND_SUBJECT}  Upload file test subject
-
+${METHODOLOGY_NAME}  newMethodology-%{RUN_IDENTIFIER}
+${METHODOLOGY_TITLE}  ${METHODOLOGY_NAME}  Title
 *** Test Cases ***
-Create publication & release
+Navigate to manage methodoligies page
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}/methodologies
+    user waits until h1 is visible  Manage methodologies
+
+Create new methodology
+    [Tags]  HappyPath
+    user clicks link  Create new methodology
+    user waits until h1 is visible  Create new methodology
+    user enters text into element  id:createMethodologyForm-title  ${METHODOLOGY_NAME}
+    user clicks button  Create methodology
+
+Add methodology content
+    [Tags]  HappyPath
+    user waits until h1 is visible  ${METHODOLOGY_NAME}
+    user clicks link  Manage content
+    user clicks button  Add new section
+    user waits until page contains button  New section
+    user clicks button  New section
+    user scrolls down  200
+    user waits until page contains button  Add text block
+    user clicks button  Add text block
+    user waits until page contains button  Edit block
+    user clicks button  Edit block
+    user scrolls down  200
```

Comment:
Are we scrolling some more because of lazy loading? The content section should already be rendered by now if I'm not mistaken

Reply:
Yep. Initially I was having lots of issues trying to find these buttons. Not sure if it was because they are lazy loaded or whether it was due to some quirk of selenium. Either way I've removed them now and the test seems to be ok now without them.